### ISSUE TITLE
O-DT-TA-1-24 Provide TWD at standstill when only TWA/TWS is available [R49]

### ISSUE THREAD

**Issue by canne** [canne/dashboard_tactics_pi#84]
Enhance the forced true wind calculation function of Tactics to support the use case where the upstream instruments or multiplexer provide the true wind calculation function but they do not provide TWD.

**Comment by canne** [comment]
The use case to this function comes from Dashboard forum [#527](http://www.cruisersforum.com/forums/f134/plugin-dashboard-44087-36.html#post2991036). In my answer [#533](http://www.cruisersforum.com/forums/f134/plugin-dashboard-44087-36.html#post2992162) I analyzed the lack of TWD being the case while the TWS/TWA were there. I never got a confirmation to my analyze from the actual boat data, but since I was able to reproduce the same behavior in my own boat, with MinixPlex multipexer doing the TWS/TWA conversion no matter what, even at standstill, I realized that I can do something for it:

The implementation is coupled to true wind calculation function. Strictly speaking it does not have to be like that, it is enough to have the AWA and the true heading and make sure that there is no drift. But I do not see how this can be controlled (this is implementation specific) elsewhere but in the Tactics's true wind calculation where everything is already there. Otherwise the user would be highly confused it will be done silently without him/her actually selecting the forced true wind calculation function of the Tactics regatta computer.

Let's do it so that when one selects Tactics Force True wind calculation and the boat happens to be at standstill and there is no TWD but there is TWA/TWS the function simply assist the upstream's true wind calculation and **only** calculates the TWD. This way, when in anchor (not tested since the boat can move a lot) or moored more steadily, one can use the Wind History instrument to observe the meteorological evolution.

As soon as the boat starts moving, the full _forcing_ comes back to the effect, like it always has been that we want speed (either SOG or StW) to calculate the TWA/TWS **and** TWD, overriding any value for those from the upstream.
